parser_test.go

```diff
@@ -213,6 +213,140 @@ func (s *Zuite) TestParser_parseExpressionsAndCheckCompute() {
 		`false && undefined`:               `false`,
 		`false && 6 / 0 round down 7 == 6`: `false`,
 
+		// gt
```

Comment:
that's a lot of tests... fine leaving as is, but generally having a smaller set of unique examples is easier to read

Reply:
I originally made these in a pretty systematic way, even though there are a lot. There were a bunch of things I wanted to check (precedence, comparisons with different scales, etc.), and then I wanted 3x tests for each (whether the left was greater, right was greater, or the sides were equal).

I think there are some I can de-dup, so I'll do that, but I don't want you to think I was nuts. 😆